01-ruby-fundamentals/README.md

```diff
@@ -0,0 +1,19 @@
+## Ruby Fundamentals
+
+1. [Welcome to Ruby](01-welcome-to-ruby.md)
+1. [Intro to Object Types](02-intro-to-object-types.md)
+1. [Data Structures](02a-data-structures.md)
+1. [Activity: Divide & Conquer Methods](02b-activity-methods-dc.md)
```

Comment:
Yes - I think I started doing this in later Readmes and didn't go back and update